test/sample-test/sample_test_launcher.py

```diff
@@ -129,7 +129,9 @@ def _compile(self):
           self._run_pipeline = raw_args['run_pipeline']
```

Comment:
Could you add some comments in the schema yaml about the run_pipeline:
1) for python samples: setting it true means submitting the pipeline.
2) for notebook samples: setting it true means telling the test infra that the sample will submit a pipeline run and the test infra will inject the experiment_name.

Reply:
In other words: run_pipeline means submitting the run for python sample and checking the results in the notebook sample.